### PR TITLE
Facilitate copy of the code

### PR DESCRIPTION
**Pull request by thiagotalma**
It is much more convenient than having to select with the mouse throughout.

### CODE REVIEW

**Comment by samdark** [comment]
Not sure it's the best way to do it.

qip shot - screen 246

qip shot - screen 247

How about selecting the text itself instead of providing it in an input box?

**Comment by thiagotalma** [comment]
The input is foolproof, works on any browser.

If you have any other suggestions just tell me that I change the PR.

**Comment by samdark** [comment]
Here's very smart way Trello handles it: https://stackoverflow.com/questions/17527870/how-does-trello-access-the-users-clipboard. So we can probably do it the same way removing any buttons and input dialogs and showing "Press CTRL+C to copy" instead.

**Comment by thiagotalma** [comment]
@samdark, very good!

File: assets/main.css [hunk 1]
```diff
@@ -250,3 +250,18 @@ body {
...
 }
 
+#clipboard-container {
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 0;
+    height: 0;
+    z-index: 100;
+    /*display: none;*/
```
Comment (samdark) [inline assets/main.css]: Remove?

**Comment by samdark** [comment]
Overall works but it adds unnecessary newlines:

```php

<?php


namespace app\models;



use Yii;



/**

 * This is the model class for table "user".

 *

 * @property integer $id
 * @property string $username
 * @property string $auth_key
```

File: assets/main.css (hunk 1 above)
Comment (thiagotalma) [inline assets/main.css]: Nop! Its required.

**Comment by thiagotalma** [comment]
Strange .... here works very well without extra spaces. (remember my first answer I gave you this PR?)

**Comment by thiagotalma** [comment]
```PHP
<?php

namespace common\models;

use Yii;

/**
 * This is the model class for table "tbl_user".
 *
 * @property integer $id
 * @property string $username
 * @property string $email
 * @property string $password_hash
 * @property string $auth_key
 * @property integer $confirmed_at
 */
class User extends \yii\db\ActiveRecord
{
    /**
     * @inheritdoc
     */
    public static function tableName()
    {
        return 'tbl_user';
    }

    /**
     * @inheritdoc
     */
    public function rules()
    {
    }
```

**Comment by samdark** [comment]
That's Chrome on windows 8. Probably browser differences...

**Comment by thiagotalma** [comment]
@samdark, I can not simulate here. You can help me test, please?

See now worked.

**Comment by samdark** [comment]
Merged with some improvements.

qip shot - screen 249

**Comment by prabowomurti** [comment]
I am using Cmd + C and Chrome. But the copied character of space become `&nbsp;` character. So I always get the syntax error message. Do you have any idea why this happens?

**Comment by cebe** [comment]
@prabowomurti could you please open a new issue for this?

**Comment by thiagotalma** [comment]
In fact, it never worked.
I ended up giving a specific solution for my needs, creating a macro using Autohotkey.

**Comment by cebe** [comment]
@thiagotalma I've been using it a few times with firefox and had no issue with it.

**Comment by thiagotalma** [comment]
@cebe So it's just a problem with Chrome...

**Pull request closed without merging**